package.json

```diff
@@ -4,58 +4,56 @@
-    "@fullhuman/postcss-purgecss": "3.1.3",
-    "@types/clipboard": "2.0.1",
-    "@types/parcel-env": "0.0.0",
-    "autoprefixer": "9.8.6",
-    "clipboard": "2.0.6",
-    "elm": "0.19.1-4",
-    "node-elm-compiler": "5.0.5",
-    "parcel-bundler": "1.12.4",
-    "parcel-plugin-static-files-copy": "2.5.0",
-    "rimraf": "3.0.2",
-    "sass": "1.32.5",
-    "typescript": "4.1.3"
+    "clipboard": "2.0.8"
   },
   "devDependencies": {
-    "axe-core": "4.1.1",
+    "@fullhuman/postcss-purgecss": "4.0.2",
+    "@parcel/transformer-elm": "2.0.0-nightly.2249",
+    "@parcel/transformer-sass": "2.0.0-nightly.627",
+    "axe-core": "4.1.3",
     "cypress": "5.6.0",
-    "cypress-axe": "0.12.0",
-    "elm-hot": "1.1.6",
+    "cypress-axe": "0.12.2",
+    "elm": "0.19.1-5",
+    "make-dir-cli": "2.0.0",
+    "ncp": "2.0.0",
+    "parcel": "2.0.0-nightly.625",
     "prettier": "2.2.1",
-    "start-server-and-test": "1.11.7",
-    "stylelint": "13.9.0",
+    "rimraf": "3.0.2",
+    "start-server-and-test": "1.12.0",
+    "stylelint": "13.12.0",
     "stylelint-a11y": "1.2.3",
     "stylelint-color-format": "1.1.0",
     "stylelint-config-prettier": "8.0.2",
     "stylelint-config-rational-order": "0.1.2",
     "stylelint-config-recommended-scss": "4.2.0",
     "stylelint-declaration-block-no-ignored-properties": "2.3.0",
-    "stylelint-declaration-strict-value": "1.7.7",
+    "stylelint-declaration-strict-value": "1.7.9",
     "stylelint-high-performance-animation": "1.5.2",
     "stylelint-order": "4.1.0",
-    "stylelint-scss": "3.18.0"
+    "stylelint-scss": "3.19.0"
   },
   "scripts": {
     "predev": "npm run clean",
     "dev": "npm run start",
     "prestart": "npm run clean",
-    "start": "parcel src/static/index.html --port 8888",
-    "build-img": "parcel build src/static/img/*",
+    "start": "npm run copy:static && parcel src/static/index.html --port 8888",
     "prebuild": "npm run clean",
-    "build": "parcel build src/static/index.html --no-minify",
-    "prebuild-prod": "npm run clean",
-    "build-prod": "parcel build src/static/index.html --no-cache --detailed-report --experimental-scope-hoisting",
-    "test": "elm-test --compiler ./node_modules/.bin/elm",
-    "test:watch": "elm-test --compiler ./node_modules/.bin/elm --watch",
+    "build": "parcel build src/static/index.html --no-optimize --detailed-report",
+    "postbuild": "npm run copy:static",
+    "prebuild:prod": "npm run clean",
+    "build:prod": "parcel build src/static/index.html --no-cache --detailed-report",
+    "postbuild:prod": "npm run copy:static",
+    "copy:static": "make-dir ./dist && ncp ./src/static/images ./dist/images",
+    "test": "elm-test",
+    "test:watch": "elm-test --watch",
     "test:cy": "start-test 8888 cypress",
     "test:cy-open": "start-test 8888 cypress:open",
     "cypress": "cypress run",
     "cypress:open": "cypress open",
-    "clean": "rimraf ./.cache ./dist",
+    "clean": "rimraf ./.parcel-cache ./.cache ./dist",
```

Comment:
One of the current, known issues with Elm + Parcel v2 - need to clean Parcel cache before a build.